index.js

```diff
@@ -133,9 +133,9 @@ async function send (ctx, path, opts = {}) {
     }
     ctx.set('Cache-Control', directives.join(','))
   }
-  ctx.type = type(path)
   ctx.body = fs.createReadStream(path)
-
+  ctx.type = ctx.res.type ? ctx.res.type : type(path);
```

Comment:
My file has no extension but i am setting content type in setHeaders function. which the module is overriding and not able to set in the response.
Its setting content type as octetstream and not the one written in the response object.